proto/inbound.proto

```diff
@@ -233,3 +240,36 @@ message GrpcRoute {
     }
   }
 }
+
+message HttpLocalRateLimit {
+  // Overall rate-limit, which all traffic coming to this server should abide
+  // to. If unset no overall limit is applied.
+  Limit total = 1;
+
+  // Fairness for individual identities; each separate source, grouped by
+  // identity, will have this rate-limit.
+  Limit identity = 2;
+
+  // Limit overrides for traffic from a specific sources.
+  repeated Override overrides = 3;
+
+  message Override {
+    Limit limit = 1;
+
+    ClientIdentities clients = 2;
+
+    message ClientIdentities {
+      // A list of literal identities.
+      repeated Identity identities = 1;
+
+      // A list of identity suffixes.
+      //
+      // If this contains an empty suffix, all identities are matched.
+      repeated IdentitySuffix suffixes = 2;
```

Comment:
We're not planning on shipping suffix matches in the first version so it can be omitted.

Reply:
Ok, I guess I'll still leave the `ClientIdentities` wrapper to leave room to add this back again in the future.